Docs/doc.css

```diff
@@ -7,63 +7,75 @@
         color: var(--prime-blue);
         font-weight: 600;
     }
-    
+
     .flex-row {
         width: 90%;
         margin: 0 auto;
     }
-    
+
+    .ham-menu {
+        display: none;
+    }
+
+    #mode-switch {
+        cursor: pointer !important;
```

Comment:
use of `!important`,

Reply:
Thankyou @shivampandey0 . Will make the change